text/0000-fabric-website.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Fabric_Website
+nav_order: 3
+---
+
+- Feature Name: Fabric Website
+- Start Date: (fill me in with today's  date, 2020-08-14)
+- RFC PR: (leave this empty)
+- Fabric Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC details the proposal for a new website for using Hyperledger Fabric. This website would serve as a "front door" for the project, and collate the various resources available to help users get started with Fabric.
```

Comment:
This is a very low level comment, but, can we perhaps make the [header](https://warm-reaches-04614.herokuapp.com/static/media/Fabric%20website%20header.dd2503aa.png) be blue instead of red? Red is a very opinionated color that represents several nationalities and certain ideologies while blue is rather neutral.

Reply:
The reason that we went with the red was to tie in with the [Fabric logo](https://www.hyperledger.org/wp-content/uploads/2018/04/fabric-logo.png), which also incorporates red.